containers/DvmState.ts

```diff
@@ -39,25 +36,29 @@ const useContractState = () => {
   // get state from EMP
   const queryState = async () => {
     if (
-      dvm !== null &&
+      votingContract !== null &&
+      storeContract !== null &&
+      collAddress !== null &&
       priceIdentifier !== null &&
       expirationTimestamp !== null
     ) {
       const res = await Promise.all([
```

Comment:
Small stylistic comment (I know this code pre-dated this PR). Would this be easier to read if it were:
```
const [hasPriceResult, finalFeeResult] = Promise.all...
```

Reply:
ye, can change it to that but perhaps in a separate cleaning PR. I have created an issue to keep track of this point #106

Reply:
I went ahead and added it to this PR anyways as it's pretty small